src/storm/solver/MathsatSmtSolver.cpp

```diff
@@ -113,7 +113,7 @@ std::string MathsatSmtSolver::MathsatModelReference::toString() const {
 
 #endif
 
-MathsatSmtSolver::MathsatSmtSolver(storm::expressions::ExpressionManager& manager, Options const& options)
+MathsatSmtSolver::MathsatSmtSolver(storm::expressions::ExpressionManager& manager, [[maybe_unused]] Options const& options)
```

Comment:
In all these MathSat bindings, I see the risk that the maybe unused appears as the function may not do anything. However, I am afraid that this surpresses potentially very useful warnings when a user has mathsat. I am okay with committing, but not convinced that this is the best thing to do in these cases.

Reply:
I don't really know my way around mathsat. I added the maybe_unused based on the structure of the file, i.e. whenever the unused parameter warning appeared but the parameter was used in an #ifdef block. I can totally readjust that in whichever way it makes most sense / is desired, just say the word :)

Reply:
@tquatmann @jipspel Your input would be appreciated.

Reply:
I think it would be great if there just is no `MathsatSmtSolver` class when Storm is build without `Mathsat`. I'm not sure how easily this can be implemented, though.